Xwt.XamMac/Xwt.Mac/PopoverBackend.cs

```diff
@@ -200,6 +200,22 @@ public void DisableEvent (object eventId)
 			}
 		}
 
+		public void Show (Popover.Position orientation, object reference, Rectangle positionRect, Widget child)
+		{
+			var refView = reference as NSView;
+			if (refView == null)
+				return;
+
+			// If the rect is empty, the coordinates of the rect will be ignored.
+			// Set the width and height, for the positioning to function correctly.
```

Comment:
If the rectangle is empty, Xwt should use the target widget bounds. At least this is how its supposed to be and its the default Cocoa behavior. If you need to specify coordinates relative to the target widget, you should use a non-empty rectangle around the coords and the popup should align to the center of the specified edge of this rectangle.